### DISCUSSION TITLE
FirebirdSQL support

### DISCUSSION THREAD

**Discussion by NetLancer** [directus/directus#18051] [Archive]
Its so weird these days none of the known modern data platforms | frameworks seem to support firebirdSql although firebird has pretty characteristics & is still maintained and keeps growing. And my last resort in investigating the support problem happened to be directus. 
Love directus, many thanks to its inventors and maintainers, great job (!)
So,  the question is if there are plans to include firebirdSQL into its list too ?

**Comment by rijkvanzanten** [comment]
> So, the question is if there are plans to include firebirdSQL into its list too ?

Really depends! It doesn't seem to be a drop-in replacement for one of the "major" vendors (eg mysql/postgres/mssql), which makes implementation a little trickier. I also have to admit that I've never heard of FirebirdSQL and that you're the first one to ask about it in the 6-7 years I've been working on this platform, so in terms of community interest and engagement, it currently scores pretty low 😬

**Reply by NetLancer** [reply]
The main reason for me to dig around for lighter weight alternative RDBMs was a really old PC i've had recently and was making attempts to make some use of it while doing some minimal fullstack :)). 
I no longer have it, but the idea of some less consuming yet still maintained SQL system seems somewhat appealing to me

**Reply by rijkvanzanten** [reply]
Yeah! Sounds very interesting 🙂 That being said, it's of course a little hard to defend spending the time building and maintaining a whole new database support layer to just power one project on somebody's old PC haha 😁
